library-app/configuration/config.go

```diff
@@ -49,23 +55,48 @@ func (config Config) SetLogSeverity() error {
 	return nil
 }
 
-func (config Config) LogDebugConfigAttributes() {
-	log.Debug("[LoadConfig] Configuration:")
-	log.Debug("[LoadConfig]		port: " + config.Port)
-	log.Debug("[LoadConfig]		log format: " + config.LogFormat)
-	log.Debug("[LoadConfig]		log severity: " + config.LogSeverity)
+func (config Config) logDebugConfigAttributes() {
+	log.Debug("[logDebugConfigAttributes] Configuration:")
+	log.Debug("[logDebugConfigAttributes]		port: " + config.Port)
+	log.Debug("[logDebugConfigAttributes]		log format: " + config.LogFormat)
+	log.Debug("[logDebugConfigAttributes]		log severity: " + config.LogSeverity)
+}
+
+func validateDbEnvVars(config *Config) error {
```

Comment:
In Go convention for abbreviations is to be spelled with capitals. For example ID, DB, UUID, GUID, ...
More info on naming conventions in Go here: https://google.github.io/styleguide/go/decisions.html